cmd/builder/internal/command.go

```diff
@@ -83,10 +83,8 @@ func Command() (*cobra.Command, error) {
 
 func initConfig() error {
 	cfg.Logger.Info("OpenTelemetry Collector distribution builder", zap.String("version", version), zap.String("date", date))
-	// use the default path if there is no config file being specified
-	if cfgFile == "" {
-		cfgFile = "$HOME/.otelcol-builder.yaml"
-	}
+
+	cfg.Logger.Info("Using config file", zap.String("path", cfgFile))
```

Comment:
I think leaving this log at the end makes sense.  If we move it to the top we could have the situation where we print "Using config file <filename>" and then immediately print "failed to load config file <filename>", which seems confusing.

Reply:
Reverted this change.